Sources/BraintreeCore/Authorization/BTClientToken.swift

```diff
@@ -15,12 +15,15 @@ import Foundation
 
     /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
     /// The extracted authorization fingerprint
-    public let authorizationFingerprint: String
+    public let bearer: String
 
     /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
     /// The extracted configURL
     public let configURL: URL
 
+    /// :nodoc:
+    public let type = AuthorizationType.clientToken
```

Comment:
I can go either way! It's kind of nice to be able to reference this type like `apiClient.authorization.type == .clientToken`

Down for whatever folks think is the most clear

Reply:
yes I agree since the property in `BTAPIClient` is `authorization`, it reads easier to me just to have `type` thatcher than `apiClient.authorization.authorizationType`

Reply:
Gotcha, missed that it was nested within the `authorization`, that make sense